src/app/teams/teams.component.ts

```diff
@@ -71,7 +71,13 @@ export class TeamsComponent implements OnInit, OnDestroy {
   }
 
   private subscribePayloads(): void {
-    this.payload$ = this.organizationService.selectedOrganization$.pipe(
+    this.payload$ = this.organizationService.getAllMemoized().pipe(
+      switchMap((orgs) => {
+        if (orgs.length === 0) {
+          throw new Error('no organizations available');
```

Comment:
Wird dieser Error irgendwo gehandelt und dem User angezeigt?

Reply:
Ja, aber mit einer anderen Fehlermeldung: `Teams could not be loaded.`